src/build-info.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Basic build info to help identify what version of the app is running.
+ * This file will be re-generated in CI with real values.
+ *
+ * @see ../scripts/ci.sh
+ */
+export const BUILD_INFO = {
+	'ShippedBy-revision': 'DEV',
+	'ShippedBy-buildNumber': 'DEV',
```

Comment:
Ideally this would be dot notation (`ShippedBy.revision`), however `ShippedBy` is already indexed as a string and we can't (easily) change this to an object in elasticsearch as seen below:

```
Filter | index{_index=logstash-dev-tools-2022.06.23, _type=_doc}{metaClass={metaClass={metaClass={response={index={_index=logstash-dev-tools-2022.06.23, _type=_doc, _id=cBSpkIEBivlYiyyTW3Zh, error={reason=Could not dynamically add mapping for field [ShippedBy.buildNumber]. Existing mapping for [ShippedBy] must be of type object but found [text]., type=mapper_parsing_exception}, status=400}}, action=["index", {:_id=>nil, :_index=>"logstash-dev-tools-2022.06.23", :routing=>nil, :_type=>"_doc"}, #<LogStash::Event:0x2a8bab40>]}}}}
```